ansible_ai_connect/users/pipeline.py

```diff
@@ -84,6 +84,17 @@ def github_get_username(strategy, details, backend, user=None, *args, **kwargs):
         return get_username(strategy, details, backend, user, *args, **kwargs)
 
 
+def is_rh_user(user):
```

Comment:
Unlike with the current user.rh_employee, the email may not have been validated. Anyone from outside Red Hat can an Org and manually create a user with a @redhat.com address.